app/src/main/res/drawable/ic_send_green_24dp.xml

```diff
@@ -0,0 +1,9 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
There is no need to upload the same vector in a different color, because we can use the `setTint()` method to change the color of the original icon.